src/components/elements/inputs/input.js

```diff
@@ -27,7 +27,7 @@ export default class Input extends Component {
 
     return (
       <div
-        className={cx('ct-element', `ct-${this.props.property.type}-element`)}>
+        className={cx('ct-element', `ct-${this.props.property.type.replace(/\[\]/,'-list')}-element`)}>
```

Comment:
We may want to write something about this change since it will break styling in existing applications if someone targeted the old format.